build-tools/xa-prep-tasks/Xamarin.Android.BuildTools.PrepTasks/ProcessLogcatTiming.Shared.cs

```diff
@@ -0,0 +1,121 @@
+//
```

Comment:
…odd that this file wasn't treated as a file rename from `ProcessLogcatTiming.cs`…

Reply:
:shrug: